data/mappings/keyboard_aliases.hjson

```diff
@@ -2094,11 +2094,11 @@
     "rmi_kb/wete": {
         "target": "rmi_kb/wete/v2"
     },
-    "rookiebwoy/late9": {
```

Comment:
These shouldn't be changed, as it's basically removing them.

The new entries should be added and thes updated.

Reply:
But as mentioned earlier, from PR checklist

> data/mappings/keyboard_aliases.hjson must be updated to reflect the move, so users with pre-created configurator keymap.json files continue to detect the correct keyboard

That's why I updated all the deprecated paths and aliases.

I should update only the target? Or I should just append at the end the new paths? Sorry but I don't understand what I have to do. The only documentation about this file I've found is the issue #24834

Reply:
Yes, but you're removing older entries, which will prevent them from working for the older names.  Changes should be in addition to, not replacing previous entries.

Reply:
I have reverted the previous changes and added the new entries as suggested. Now lint and build doesn't work because 
> duplicate key: 'late9/rev1'

I have clearly understood something wrong. Can you please provide me more hints?